Shared/Confirm-ExchangeManagementShell.ps1

```diff
@@ -0,0 +1,20 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+<#
+.SYNOPSIS
+    Returns True if we are running inside Exchange Management Shell, and False otherwise.
```

Comment:
If we aren't going to have Error Monitoring handling logic added to this, then we should call that out in the comments as well.

Other Shared scripts that handle an error should be passing in the ability to do a custom `$CatchActionFunction`.

Reply:
I'm not sure it make sense to add a comment calling out that a parameter is not present. I mean, look at the parameters. There is no -CatchActionFunction.

Reply:
Fair enough